### ISSUE TITLE
ability to set stock/growth rate for multiplayer games?

### ISSUE THREAD

**Issue by FelixSchwarz** [nicymike/serflings#137]
Is it possible to add set the stock for each player in a multiplayer game?

**Comment by nicymike** [comment]
It is currently not possible to change values for multiplayer.

You may create a two player game in the original settlers with custom values and save it. Afterwards you can load it in Serflings to play it.

**Comment by FelixSchwarz** [comment]
Thank you - that is a neat idea. I just tried that but unfortunately I can not load the save in serflings. Once I copy the new save game from the original game into the servlings directly (overwriting a save game from serflings so I don't have to modify ARCHIV.DS), I can not load  saved game anymore (at all).

I'm probably missing a step I guess?

**Comment by nicymike** [comment]
Do you get any error messages?

**Comment by FelixSchwarz** [comment]
no, just all entries are "greyed" out and not clickable. Nothing on the console though.

**Comment by nicymike** [comment]
Ok, no idea how this could happen. Can you attack a zip file with the save and archive files included?

**Comment by FelixSchwarz** [comment]
sure, will do this later today.

**Comment by FelixSchwarz** [comment]
Ok, so here is my `ARCHIV.DS` and `SAVE0.DS`: [saves.zip](https://github.com/user-attachments/files/17879910/saves.zip)

I'm using `serflings-2.0.0.jar`, MD5SUMS: [MD5SUM.txt](https://github.com/user-attachments/files/17879912/MD5SUM.txt)

And this is how it looks in-game:
![loading-screen](https://github.com/user-attachments/assets/a9348c75-cfd6-4607-87a5-041582b774f0)

**Comment by nicymike** [comment]
Oh I got it. Because you created a multiplayer game you have to load it while creating a multiplayer game. The standard load button is just for loading singleplayer games.
But you may also load singleplayer games with multiplayer which will create a coop game where you both play the first player.

![image](https://github.com/user-attachments/assets/846116eb-8ebc-42b3-b286-7d6f6965578b)

**Comment by FelixSchwarz** [comment]
Ah, got it. Thank you.
